libs/core/src/main/java/com/sourcefuse/jarc/core/commonutils/CommonUtils.java

```diff
@@ -0,0 +1,28 @@
+package com.sourcefuse.jarc.core.commonutils;
+
+import java.beans.PropertyDescriptor;
+import java.util.HashSet;
+import java.util.Set;
+import org.springframework.beans.BeanWrapper;
+import org.springframework.beans.BeanWrapperImpl;
+import org.springframework.stereotype.Component;
+
+@Component
+public final class CommonUtils {
+
+  private CommonUtils() {}
+
+  public static String[] getNullPropertyNames(Object source) {
+    final BeanWrapper src = new BeanWrapperImpl(source);
+    PropertyDescriptor[] pds = src.getPropertyDescriptors();
+    Set<String> emptyNames = new HashSet<>();
+    for (PropertyDescriptor pd : pds) {
+      Object srcValue = src.getPropertyValue(pd.getName());
```

Comment:
For below scenario will this work -
consider have model Role - String Id; String Name (Nullable = false) ; String Permissions (Nullable = true);
if database has record  for Role with id abc and we are trying to update the same role 
with following body
body-{ Name: "new Name", permission: null}
DB record for role abc - { id :"abc", Name: "role", Permissions:"Get"}

does permission will be set to null or not? can you check how this works on loopback